framework/tests/aliasing/UniqueConstructorTest.java

```diff
@@ -0,0 +1,24 @@
+import org.checkerframework.common.aliasing.qual.Unique;
+
+public class UniqueConstructorTest {
+    // No need for @SuppressWarnings()
+    @Unique UniqueConstructorTest() {
+        // Doesn't raise unique.leaked error since it's a top parent class (@Unique Object subclass
+        // by default)
+    }
+}
+
+class ParentClass extends UniqueConstructorTest {
+
+    ParentClass() {
+        // Doesn't raise unique.leaked error since it's a non unique class with a unique parent
```

Comment:
`non-unique`

Reply:
I have made the correction in the comment